application/src/main/scala/com/azavea/franklin/commands/ApiOptions.scala

```diff
@@ -1,90 +1,100 @@
 
   private val apiHostHelp =
     s"Hostname Franklin is hosted it (e.g. localhost). Default: '$apiHostDefault'."
 
-  val apiHost = Opts
-    .option[String]("api-host", help = apiHostHelp) orElse
-    Opts.env[String]("API_HOST", help = apiHostHelp) withDefault (apiHostDefault)
+  val apiHost = Opts.option[String]("api-host", help = apiHostHelp) orElse Opts
+    .env[String]("API_HOST", help = apiHostHelp) withDefault (apiHostDefault)
 
   private val apiPathHelp = "Path component for root of Franklin instance (e.g. /stac/api)."
 
-  val apiPath = Opts
-    .option[String](
-      "api-path",
-      help = apiPathHelp
-    ) orElse
-    Opts.env[String]("API_PATH", help = apiPathHelp) orNone
+  val apiPath = (Opts.option[String](
+    "api-path",
+    help = apiPathHelp
+  ) orElse Opts.env[String]("API_PATH", help = apiPathHelp)) orNone
 
   private val apiSchemeDefault = "http"
 
   private val apiSchemeHelp =
     s"Scheme server is exposed to end users with. Default: '$apiSchemeDefault'."
 
-  val apiScheme =
-    Opts
-      .option[String]("api-scheme", help = apiSchemeHelp) orElse
-      Opts.env[String]("API_SCHEME", help = apiSchemeHelp) withDefault ("http")
-  validate("Scheme must be either 'http' or 'https'")(s => (s == "http" || s == "https"))
+  val apiScheme = (Opts
+    .option[String]("api-scheme", help = apiSchemeHelp) orElse Opts
+    .env[String]("API_SCHEME", help = apiSchemeHelp))
+    .withDefault("http")
+    .validate(
+      "Scheme must be either 'http' or 'https'"
+    )(s => (s == "http" || s == "https"))
 
   private val defaultLimitDefault = NonNegInt(30)
 
   private val defaultLimitHelp =
     s"Default limit for items returned in paginated responses. Default: '$defaultLimitDefault'."
 
-  private val defaultLimit =
-    Opts
-      .option[NonNegInt]("default-limit", help = defaultLimitHelp) orElse
-      Opts.env[NonNegInt]("API_DEFAULT_LIMIT", help = defaultLimitHelp) withDefault (defaultLimitDefault)
+  private val defaultLimit = Opts
+    .option[NonNegInt]("default-limit", help = defaultLimitHelp) orElse Opts
+    .env[NonNegInt]("API_DEFAULT_LIMIT", help = defaultLimitHelp) withDefault (defaultLimitDefault)
 
   private val enableTransactionsHelp =
     "Whether to respond to transaction requests, like adding or updating items. Default: 'false'."
 
-  private val enableTransactions =
-    Opts
-      .flag(
-        "with-transactions",
-        help = enableTransactionsHelp
-      ) orElse
-      Opts.env[Boolean]("API_WITH_TRANSACTIONS", help = enableTransactionsHelp) orFalse
+  private val enableTransactions = Opts
+    .flag(
+      "with-transactions",
+      help = enableTransactionsHelp
+    )
+    .orFalse orElse Opts
+    .env[String]("API_WITH_TRANSACTIONS", help = enableTransactionsHelp, metavar = "true||false")
+    .mapValidated { s =>
+      if (Try(s.toBoolean).getOrElse(false)) Validated.valid(s.toBoolean)
+      else Validated.invalidNel("Value must be Boolean")
```

Comment:
I think this won't do quite what you want --

If I pass `true`, the `Try` succeeds, and I get `true` out, but if I pass `false`... the `Try` succeeds, but it's `false`, so the conditional goes to the `else` branch and I'm told my value must be a `Boolean`.

Instead, you can use [`Validated.fromTry`](https://github.com/typelevel/cats/blob/master/core/src/main/scala/cats/data/Validated.scala#L1104-L1111) to convert the `Try` directly and then `leftMap` the result of that into the nice error string

```suggestion
      Validated.fromTry(Try(s.toBoolean)).leftMap(_ => s"I expected to find a value I could convert to a Boolean, but I got $s")
```

Reply:
e115d92